prog.go

```diff
@@ -35,14 +35,54 @@ const (
 // verifier log.
 const DefaultVerifierLogSize = 64 * 1024
 
+// LogLevel controls the verbosity of the kernel's eBPF program verifier.
+// These constants can be used for the ProgramOptions.LogLevel field.
+type LogLevel uint32
```

Comment:
Nit: how about defining this in sys and using it from the attr?

Reply:
What about adding stringer for LogLevel?

Reply:
Sounds good!